src/main/java/com/powsybl/openloadflow/ac/outerloop/IncrementalShuntVoltageControlOuterLoop.java

```diff
@@ -86,7 +88,27 @@ private static int[] createControllerShuntIndex(LfNetwork network, List<LfShunt>
         private static DenseMatrix calculateSensitivityValues(List<LfShunt> controllerShunts, int[] controllerShuntIndex,
                                                               EquationSystem<AcVariableType, AcEquationType> equationSystem,
                                                               JacobianMatrix<AcVariableType, AcEquationType> j) {
-            DenseMatrix rhs = new DenseMatrix(equationSystem.getIndex().getSortedEquationsToSolve().size(), controllerShunts.size());
+            int nRows = equationSystem.getIndex().getSortedEquationsToSolve().size();
+            int nColumns = controllerShunts.size();
+
+            if (nColumns == 1) {
```

Comment:
What is the purpose of this special case ? It does not optimize anything. Calling solve on a vector or on a matrix with one column is exactly the same.

Reply:
That's true, just fixed it